handbook/ops/bizops/amplitude.md

```diff
@@ -0,0 +1,29 @@
+## How do I get started?
+1. [Understand the data](#data)
+1. Amplitude documentation for [building analyses](https://help.amplitude.com/hc/en-us/categories/360003165371-Build-and-share-your-analysis)
+1. Ask BizOps!
+
+## Why Amplitude?
+Amplitude is a specialized product analytics tool that specializes in turning event data into actionable insights and dashboards. 
+
+### Why aren’t we using Looker for this?
+Looker is very flexible in that we have the ability to set it up for any purpose. But this also means that to provide a great self-service experience for what we want from a product analytics tool, this would take a LOT of work. We get these capabilities out of the box with Amplitude. 
+
+### What is in Looker vs. Amplitude?
+
+Anything not based directly on analyzing events is in Looker. This includes pings from on-prem instances, anything we get from the Cloud Postgres database, and any data from third-parties tools (such as Google Analytics and Salesforce). 
```

Comment:
I'm not really sure what you mean by "anything not based directly on analyzing events is in Looker." My interpretation of an "event" may be wrong, but don't we look at things like searches submitted and code intel hovers in Looker? I think of those as an analysis of events (for better or worse). The way I was interpreting Looker vs Amplitude is that (for now) anything related to the analysis of drivers of retention/adoption/engagement is best done in Amplitude, but other analysis is done in Looker. Is that right?

Reply:
We do look at events in Looker but after they're aggregated in pings - I'll make that more clear! 

And yeah Retention/engagement is best done in Amplitude because they handle the events out of the box in those specific ways, but over the past day I've started to look at it more like self-service for any product usage question, like this [breakdown between clicking def and ref](https://analytics.amplitude.com/sourcegraph/chart/3lcp9ju?source=workspace). So you are right, but I've also changed my thinking quite a bit in the last 24 hours 😄